src/api/constants/index.ts

```diff
@@ -33,6 +33,21 @@ const DATE_5_MINUTES_FROM_NOW = () => {
   return future;
 };
 
+/**
+ * DATE_30_MINUTES_FROM_NOW
+ * Generate a date that is 30 minutes from now
+ * @returns {Date}
+ */
+const DATE_30_MINUTES_FROM_NOW = () => {
+  const now = new Date();
+
+  const minutes = 30;
+
+  const future = new Date(now.getTime() + minutes * 60000);
```

Comment:
Can we add BODMAS to this please `now.getTime() + (minutes * 60000))`

Reply:
We could, but linting flags this as being pointless and asks to remove it.

Reply:
Haha okay we can ignore it then.